composer.json

```diff
@@ -41,7 +41,7 @@
         ]
     },
     "require-dev": {
-        "friendsofphp/php-cs-fixer": "2.16.*",
+        "friendsofphp/php-cs-fixer": "~2.16.1",
```

Comment:
Note: actually 2.16.0 does not work with PHP 7.4. So we need to specify to use at least 2.16.1 and upwards (2.16.2 2.16.3 ...)